src/AssetsTransferApi.ts

```diff
@@ -397,4 +397,44 @@ export class AssetsTransferApi {
 			return AssetType.Foreign;
 		}
 	}
+	/**
+	 * Decodes the hex of an extrinsic into a string readable format
+	 *
+	 * @param encodedTransaction the hex of an extrinsic tx
+	 * @param format The format the tx is in
+	 */
+	public decodeExtrinsic<T extends Format>(
+		encodedTransaction: string,
+		format: T
+	): string {
+		const { _api } = this;
+		const fmt = format ? format : 'payload';
+
+		if (fmt === 'payload') {
+			const extrinsicPayload = _api.registry.createType(
+				'ExtrinsicPayload',
+				encodedTransaction,
+				{
+					version: 4,
+				}
+			);
+
+			const extrinsicMethodInfo = extrinsicPayload.method.toHuman() as string;
```

Comment:
Why do we do `toHuman()` here but `toString` in the other two formats? Any way to keep them all the same?

Reply:
the tx method field for the extrinsic payload is of type `Bytes` so I called `toHuman` in order to first turn then into an object. We can call toHuman()?.toString() but ultimately we can't directly `toString` from `Bytes` since we essentially get the hex and not the human readable object. Ill update this to call `toString` after in case thats more clear.